pkg/api/api.go

```diff
@@ -196,13 +196,17 @@ func (a *API) specificNodesHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	// TODO: these nodes don't send base target value at the moment
+	// TODO: these nodes don't send a generator at the moment
+	// TODO: support the generator field
 	stateHashEvent := entities.NewStateHashEvent(
 		statement.Node,
 		zeroTS,
 		statement.Version,
 		statement.Height,
 		statehash,
 		zeroBT,
+		&statehash.BlockID,
+		nil,
```

Comment:
If privates nodes are not going to send us the generator info, let's put an empty address here instead of nil